src/django/requirements.txt

```
@@ -3,6 +3,7 @@ boto3==1.9.92
 coreapi==2.3.3
 dedupe==1.9.4
 django-amazon-ses==2.0.0
+git+git://github.com/azavea/django-ecsmanage@feature/jrb/fix-python-37#django-ecsmanage
```

Reminder after https://github.com/azavea/django-ecsmanage/pull/9 is merged.

```suggestion
django-ecsmanage==1.0.1
```